src/test/java/medic/gateway/alert/SmsSenderTest.java

```diff
@@ -70,8 +70,8 @@ public void sendUnsentSmses_shouldPassUnsentMessagesToSmsManager() {
 		for(Status s : Status.values()) {
 			// create a WoMessage with each status
 			db.insert("wo_message",
-					cols("_id",     "status", "last_action", "_to", "content"),
-					vals("id-" + s, s,        0,             "+1",  "testing: " + s));
+					cols("_id",     "status", "last_action", "_to", "content", "retries"),
+					vals("id-" + s, s,        0,             "+1",  "testing: " + s, 0));
```

Comment:
Is it possible to add some tests here to check if the retry mechanism is successful:
- skip retry when it's not time
- do retry when it's time
- retry interval calculation for multiple retries
- no retry when exceeding max nbr.

Reply:
Thanks! Done